multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -45,18 +142,470 @@ type ServiceExportReconciler struct {
 
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
-// For ServiceExport controller, it watches events of ServiceExport resource, and also
-// Endpoints/Services resource if they have the label 'antrea.io/multi-cluster'.
-// It will create/update/remove ResourceExport resource in leader cluster
-// for corresponding ServiceExport in member cluster.
+// For ServiceExport controller, it watches events of ServiceExport resource,
+// and also Endpoints/Services resource. It will create/update/remove ResourceExport
+// resource in leader cluster for corresponding ServiceExport in member cluster.
```

Comment:
In general, I feel you should add articles before a noun, or use the plural form.

Reply:
done